### PR TITLE
magit-todos: supress the jT unbound warning

### PR DESCRIPTION
**Pull request by condy0919**

### CODE REVIEW

File: modes/magit-todos/evil-collection-magit-todos.el [hunk 1]
```diff
@@ -39,6 +39,11 @@
   (evil-collection-define-key 'normal 'magit-status-mode-map
     "gT" (and (bound-and-true-p magit-todos-mode) 'magit-todos-jump-to-todos)))
 
+(defun evil-collection-magit-todos-supress-warning (func &rest args)
+  "Supress the jT keybinding warning."
+  (cl-letf* (((symbol-function 'message)
+              (lambda (&rest _) nil)))
```
Comment (Ambrevar) [inline modes/magit-todos/evil-collection-magit-todos.el:45]: Couldn't we replace the lambda by `#'ignore` and `(symbol-function 'message)` by `#'message`?

Comment (condy0919) [reply modes/magit-todos/evil-collection-magit-todos.el]: It seems that `(symbol-function 'message)` is inreplaceable from [this answer](https://stackoverflow.com/questions/39550578/in-emacs-what-is-the-difference-between-cl-flet-and-cl-letf)

**Comment by Ambrevar** [comment]
My bad, you are right! :)

**Comment by condy0919** [comment]
ready for mereg?

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**